crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -1576,10 +1601,122 @@ type MoveData =
   | { String:  string }
   | { Vector:  [MoveData] }
   | { Option:   MoveData? }
-  | { Struct:  [{ name: string, value: MoveData }] }
+  | { Struct:  [{ name: string , value: MoveData }] }
+  | { Variant: {
+      name: string,
+      fields: [{ name: string, value: MoveData }],
+  }
 """
 scalar MoveData
 
+type MoveDatatype implements IMoveDatatype {
+	module: MoveModule!
+	name: String!
+	abilities: [MoveAbility!]
+	typeParameters: [MoveStructTypeParameter!]
+	asMoveEnum: MoveEnum
+	asMoveStruct: MoveStruct
+}
```

Comment:
We should add some doc comments to the type that implements this, so that it gets some docs.

Reply:
ah! I was wondering how to get docs on this! Will definitely do this.